docs/pages/get-started/faq.md

```diff
@@ -384,109 +411,148 @@ Clients communicate with XMTP nodes through a gRPC (or JSON/HTTP) message API.
 
 XMTP provides perceptibly real-time message delivery and retrieval. The network does not provide service level guarantees.
 
-### Is there a way to get a list of all current XMTP-enabled wallets?
+## Fees
 
-XMTP doesn't provide a direct method to fetch all XMTP-enabled wallet addresses in bulk. You might consider using a third-party service to do so instead of building the functionality yourself.
+XMTP core developers and researchers are working on a specific fee model for XMTP, with the following guiding principles in mind:
 
-For example, services like [Blaze](https://www.withblaze.app/) have compiled extensive lists of XMTP addresses by querying the XMTP `canMessage` API over time at a rate that avoids hitting the limits.
+- Infrastructure costs for the network must remain low even when decentralized, and comparable to the costs for an equivalent centralized messaging service.
+- There must be a sizable free tier for most end users, with the low cost per-user subsidized by the largest apps and bulk senders.
+- There must be a low "take rate": the biggest driver of cost must be infrastructure costs, with any remaining cost returned to the network.
 
-### Does XMTP support group chat?
+Have questions or feedback about the fee model for XMTP? See [XMTP fees: Guiding principles and FAQ](https://community.xmtp.org/t/xmtp-fees-guiding-principles-and-faq/795) in the XMTP Community Forums.
 
-Yes. XMTP V3 supports group chat in React Native, Android, iOS, and Node.js SDKs.
+## Security
 
-:::info[For more info]
-See [Build group chat with MLS and XMTP](/groups/build-group-chat)
-:::
+### Has XMTP undergone a security audit?
 
-## Fees
+A security audit of [LibXMTP](https://github.com/xmtp/libxmtp) and its use of Messaging Layer Security (MLS) will begin in Q4 2024. The audit will be performed by [NCC Group](https://www.nccgroup.com/).
 
-### Who pays to keep the network running?
+The [XMTP JavaScript SDK](https://github.com/xmtp/xmtp-js) (`xmtp-js`) was verified by [CertiK](https://www.certik.com/company/about) on Feb 6, 2023. Read their [XMTP security assessment](https://raw.githubusercontent.com/xmtp/docs-xmtp-org/main/docs/pages/img/REP-final-20230207T000355Z.pdf).
 
-Ephemera is committed to keeping the network running. Future incentive mechanisms will enable the network to run autonomously of Ephemera.
+### How does XMTP establish a secure and fraud-proof relationship between identities?
```

Comment:
LGTM, the only comment would be to make sure that app developers know that this is mostly handled by the SDK under the hood